client/src/components/More.jsx

```diff
@@ -274,6 +283,7 @@ const More = ({ setSubscribedNumber, unsubscribeNumber, subscribedNumber, clearR
               className={classes.buttons}
               onClick={() => {
                 handleClose();
+                setReminderStatus(true);
```

Comment:
is the aim to show the dialogue again if the user sings out and then in? if so it didn't work

Reply:
I think it's just if you haven't completed a survey on that calendar date, right?

Reply:
Oh yeah! if it is only shown again if you haven't completed a survey 'today' then it makes sense